assets/scss/learndash-styling.scss

```diff
@@ -512,7 +542,7 @@
 	  margin-bottom  : nhsuk-spacing(6);
 	  @extend .nhsuk-card;
 	  position       : relative;
-	  padding-bottom : 120px;
+	  padding-bottom : 95px;
```

Comment:
can we get rid of the hard coded value? (Again, acknowledge the hard coded value isnt this PR, but...)